packages/cloud_firestore/cloud_firestore/test/cloud_firestore_e2e.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:e2e/e2e.dart';
```

Comment:
move this file to 
`cloud_firestore/cloud_firestore/exmaple/test_driver/cloud_firestore_e2e.dart`
 to fix CI